jobs/build/ocp3/Jenkinsfile

```diff
@@ -732,13 +732,13 @@ node {
 
 
             // Push the building puddle out to the correct directory on the mirrors (e.g. online-int, online-stg, or enterprise-X.Y)
-            buildlib.invoke_on_rcm_guest("3.11-plashet-to-rcm-guest.sh", SYMLINK_NAME, NEW_FULL_VERSION, BUILD_MODE)
+            buildlib.invoke_on_ocp_artifacts("3.11-plashet-to-ocp-artifacts.sh", SYMLINK_NAME, NEW_FULL_VERSION, BUILD_MODE)
 
-            // 3.11-plashet-to-rcm-guest.sh sets up a different puddle name on rcm-guest and the mirrors
+            // 3.11-plashet-to-ocp-artifacts.sh sets up a different puddle name on rcm-guest and the mirrors
             PLASHET = "v${NEW_FULL_VERSION}_${PLASHET}"
             final mirror_url = "https://mirror.openshift.com/enterprise/enterprise-${params.BUILD_VERSION}"
 
-            buildlib.invoke_on_rcm_guest("publish-oc-binary.sh", params.BUILD_VERSION, NEW_FULL_VERSION)
+            buildlib.invoke_on_ocp_artifacts("publish-oc-binary.sh", params.BUILD_VERSION, NEW_FULL_VERSION)
```

Comment:
oc binaries can be extracted from the current working directory. I don't think it is good to do that remotely on ocp-artifacts. After the binaries are extracted, they can be sync'd to mirror immediately.

Reply:
ok got it, will work on this

Reply:
e1fa2a15ab86458f3356fd7b4ecdfb33366ee302 removed [publish-oc-binary.sh](https://github.com/openshift-eng/aos-cd-jobs/pull/3574/commits/c93e14f66ec23d9a8b15977ef4c5ec5be60ca57f).